### DISCUSSION TITLE
Is it possible to make the volume bar osc vertical rather than horizontal?

### DISCUSSION THREAD

**Discussion by yousefx0** [mpv-player/mpv#14379] [Q&A]
Hello everyone!

I mapped vim keys to mpv 
H ==> seek backward
J ==> decrease volume
K ==> increase volume
L ==> seek forward

However since the layout of the volume bar osc is horizontal, sometimes I get confused and use H and L to change the volume. Is it possible to make it have a vertical layout to avoid that confusion? Thanks a lot!

**Comment by hooke007** [comment]
"volume bar" is not in osc.
So No for this question.

**Reply by yousefx0** [reply]
Apologies didn't know that.
But anyways is there a method that it can be made vertically oriented (other than osc) ?

**Reply by hooke007** [reply]
Nope.
